openedx/core/lib/courses.py

```diff
@@ -44,3 +44,26 @@ def create_course_image_thumbnail(course, dimensions):
     _content, thumb_loc = contentstore().generate_thumbnail(course_image, dimensions=dimensions)
 
     return StaticContent.serialize_asset_key_with_slash(thumb_loc)
+
+
+def additional_image_url(course, image_key):
+    """Try to look up additional image (such as hero) url for the course.  If it's not found,
+    log an error and return the dead link"""
+    if course.static_asset_path or modulestore().get_modulestore_type(course.id) == ModuleStoreEnum.Type.xml:
```

Comment:
I would prefer to see us modify the existing `course_image_url` function above and add any missing unit test coverage to that function's tests.